app/services/CheckoutService.scala

```diff
@@ -389,32 +390,63 @@ class CheckoutService(
       subscriptionDetails.withContextLogging(s"Subscription price:")
     }
 
-    def pricesMatch(displayedPrice: String, subscriptionPrice: String) = {
-      if(displayedPrice != subscriptionPrice){
-        logger.error(s"Client and server side prices divergent, we showed $displayedPrice and wanted to charge $subscriptionPrice from ${subscription.name} (${renewal.promoCode})")
-        false
+    case class ContactInfo(billto: ZuoraContact, soldto: ZuoraContact, salesforceContact: Contact)
+
+    def postRenewalSteps(subscription: Subscription[WeeklyPlanOneOff], renewal: model.Renewal, subscriptionPrice: String, salesforceContact: Contact, newTermStartDate: LocalDate) = {
+      def logException(e: Exception): Unit = {
+        logger.error(s"unexpected error in post renewal steps", e)
```

Comment:
Would be nice to use ContextLogging here so we know which subscribers didn't get an email!